### PR TITLE
Suggested updates to the diabetes example

### PR DESCRIPTION
**Pull request by hardbyte**

### CODE REVIEW

File: examples/federated_learning_with_encryption.py [hunk 1]
```diff
@@ -115,41 +116,38 @@ def get_data(n_clients):
...
 
 class Server:
-    """Hold the private key. Decrypt the average gradient"""
+    """Private key holder. Decrypts the average gradient"""
 
-    def __init__(self, key_length=1024):
-        self.pubkey, self.privkey = \
-            paillier.generate_paillier_keypair(n_length=key_length)
+    def __init__(self, key_length):
+        self.pubkey, self.privkey = paillier.generate_paillier_keypair(n_length=key_length)
```
Comment (giorgiop) [inline examples/federated_learning_with_encryption.py]: This is here to adhere with pep8 max chars per line.

File: examples/federated_learning_with_encryption.py [hunk 2]
```diff
@@ -182,34 +177,27 @@ def predict(self, X):
...
 
 
-if __name__ == '__main__':
-
-    # Learning params
-    n_iter, eta = 50, 0.01
-
-    names = ['Hospital 1', 'Hospital 2', 'Hospital 3']
-    n_clients = len(names)
+def federated_learning(n_iter, eta, n_clients, key_length):
```
Comment (giorgiop) [inline examples/federated_learning_with_encryption.py]: Do you think this help readability?

Comment (hardbyte) [inline examples/federated_learning_with_encryption.py]: I don't think it detracts readability - and it makes it easier to experiment with different number of clients and different key lengths.

Comment (giorgiop) [inline examples/federated_learning_with_encryption.py]: Yeah, that's fine

File: examples/federated_learning_with_encryption.py (hunk 1 above)
Comment (giorgiop) [inline examples/federated_learning_with_encryption.py]: Just find any way you like to please pep8 here.

Comment (hardbyte) [inline examples/federated_learning_with_encryption.py]: Sure sure. No point arguing against the standard...

**Pull request merged**